### ISSUE TITLE
Target children without key of 'children'

### ISSUE THREAD

**Issue by rtmalone** [mbraak/jqTree#363]
I am retrieving some json from the server in which the object is:

```
{ 
  graphic: something,
  label: something,
  target: something,
  screen: something,
  tricons: [...] // these are the children
}
```

Is there support to just define which key in the object has the children, instead of having to rename keys for jqTree and renaming them again when I need to post the json data from the tree back to the server?

If I've missed an answer/solution to this forgive me, but I haven't found one and I find it hard to believe someone hasn't had the same issue yet. Thanks!

**Comment by mbraak** [comment]
JqTree does not have an option for that.

The option that comes closest is the [dataFilter](http://mbraak.github.io/jqTree/#options-datafilter) option.

Also see [issue 180](https://github.com/mbraak/jqTree/issues/180).

**Comment by rtmalone** [comment]
I had been playing with dataFilter, but I am not able to get a response when using it. Here is what I've just simply tried after your suggestion:

```
  function createTriconTree(){
    console.log('run');
    $('#tricon-tree').tree({
        url:'/system/configuration',
        dataFilter: function(data){
          console.log(data);
        };,
        dragAndDrop: true,
        autoOpen: 0
    });
  }
```

The entire function is being run when the page loads. The console.log for 'run' is logged, but the one in the dataFilter doesn't fire. My server isn't logging a call for that url in the first place, so there is that. However, I can make an ajax call myself and pass the returned data to the tree which just recognizes the parent level as I previously noted in the issue with my children.

Ultimately though, it appears that even if the dataFilter was working in my case I'd still need to process that data from the server to match the label/children jqTree format. I wanted to avoid this because I'll be posting back to the server regularly, which expects the format I listed initially.

It looks like jqTree takes care of everything else I expect to need, so I might just need to bite the bullet.
